lib/kernel/src/logger_h_common.erl

```diff
@@ -396,7 +396,7 @@ do_log_to_binary(Log,Config) ->
                                      {config,FormatterConfig},
                                      {log_event,Log},
                                      {bad_return_value,String},
-                                     {catched,{C2,R2,S2}}]),
+                                     {caught,{C2,R2,S2}}]),
```

Comment:
Don't change this spelling since it is code.

Reply:
Should this variable name tally up with the `caught_call_exception` type here, like everywhere use `caught` instead of `catched`?

See https://github.com/erlang/otp/blob/master/lib/kernel/src/erpc.erl

```erlang
-type caught_call_exception() ::
        {throw, Throw :: term()}
      | {exit, {exception, Reason :: term()}}
      | {error, {exception, Reason :: term(), StackTrace :: [stack_item()]}}
      | {exit, {signal, Reason :: term()}}
      | {error, {?MODULE, Reason :: term()}}.
```

Reply:
I don't think this instance of `catched` has something to do with `erpc`.

@garazdawi Do you think it safe to change this instance of `catched` to `caught` or is the code (or documentation) that depends on it remaining `catched`?

Reply:
Should be fine to change. The code is only used when a formatter crashes and the exact keys are not documented.